packages/xz_java.rb

```diff
@@ -0,0 +1,35 @@
+require 'package'
+
+class Xz_java < Package
+  description 'A complete implementation of XZ data de/compression in pure Java.'
+  homepage 'http://tukaani.org/xz/java.html'
+  version '1.8'
+  source_url 'https://tukaani.org/xz/xz-java-1.8.zip'
+  source_sha256 '86be97adf59d19c000c5bac8c77f2fa2f84fa082d608c7ad2d07a007dce76160'
+
+  binary_url ({
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/xz_java-1.8-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+     x86_64: '5015ecd8bee78fda874c6b6cdf4576dcc3a8effe95c545e6841d2d041d916f16',
+  })
+
+  depends_on 'jdk8'
+  depends_on 'ant' => :build
+
+  def self.build
+    system "env", "JAVA_HOME=#{CREW_PREFIX}/share/jdk8", "ant"
+    system "env", "JAVA_HOME=#{CREW_PREFIX}/share/jdk8", "ant", "doc"
+    system "wget", "https://github.com/JL2210/bash-scripts/raw/c28abd3b80716637f708300e4085ed775b525844/xz-java.in"
```

Comment:
Can you please rename this file with a `.sh` extension?  The `.in` does not indicate the type of file.